core/modules/wdb/QueryAction.cc

```diff
@@ -300,6 +300,9 @@ class ChunkResourceRequest {
     proto::TaskMsg const& _msg;
 };
 
+util::Error makeError(sql::SqlErrorObject const& e) {
```

Comment:
I'll put in a comment in SqlErrorObject that we should consider this. At this point, we pay a couple lines of more code in exchange for a bit of independence and isolation.